build/rst2man.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3

```

Comment:
This needs to be `#!/usr/bin/env @PYTHON@` (and patched after configure step).

See https://github.com/skvadrik/re2c/blob/master/CMakeLists.txt#L99.

Reply:
This seems messy and unnecessary. It adds an unnecessary step to the build and it puts build artifacts into a non-build folder. You can get the same result by just putting `${PYTHON}` before each call to the script.

Reply:
Also, it doesn't work on Windows, so in all of the build scripts you have to put `${PYTHON}` in front of each use of the script anyway.